api/lib/domain/services/mail-service.js

```diff
@@ -12,97 +12,97 @@ import { config } from '../../config.js';
 const PIX_HOME_URL_FRENCH_FRANCE = `${config.domain.pix + config.domain.tldFr}`;
+const PIX_APP_URL_FRENCH_FRANCE = `${config.domain.pixApp + config.domain.tldFr}`;
+const PIX_APP_CONNECTION_URL_FRENCH_FRANCE = `${PIX_APP_URL_FRENCH_FRANCE}/connexion`;
+const PIX_ORGA_HOME_URL_FRENCH_FRANCE = `${config.domain.pixOrga + config.domain.tldFr}`;
+const PIX_CERTIF_HOME_URL_FRENCH_FRANCE = `${config.domain.pixCertif + config.domain.tldFr}`;
+const HELPDESK_FRENCH_FRANCE = 'https://support.pix.fr';
 
-const EMAIL_VERIFICATION_CODE_TAG = 'EMAIL_VERIFICATION_CODE';
-const SCO_ACCOUNT_RECOVERY_TAG = 'SCO_ACCOUNT_RECOVERY';
+// INTERNATIONAL
+const PIX_HOME_NAME_INTERNATIONAL = `pix${config.domain.tldOrg}`;
+const PIX_HOME_URL_INTERNATIONAL = {
+  en: `${config.domain.pix + config.domain.tldOrg}/en-gb/`,
+  fr: `${config.domain.pix + config.domain.tldOrg}/fr/`,
+  nl: `${config.domain.pix + config.domain.tldOrg}/nl-be/`,
+};
+const PIX_ORGA_HOME_URL_INTERNATIONAL = `${config.domain.pixOrga + config.domain.tldOrg}`;
+const PIX_CERTIF_HOME_URL_INTERNATIONAL = `${config.domain.pixCertif + config.domain.tldOrg}`;
+const PIX_APP_URL_INTERNATIONAL = `${config.domain.pixApp + config.domain.tldOrg}`;
+const PIX_APP_CONNECTION_URL_INTERNATIONAL = {
+  en: `${PIX_APP_URL_INTERNATIONAL}/connexion/?lang=en`,
+  es: `${PIX_APP_URL_INTERNATIONAL}/connexion/?lang=es`,
+  fr: `${PIX_APP_URL_INTERNATIONAL}/connexion/?lang=fr`,
+  nl: `${PIX_APP_URL_INTERNATIONAL}/connexion/?lang=nl`,
+};
+const PIX_HELPDESK_URL_INTERNATIONAL = {
+  en: 'https://support.pix.org/en/support/home',
+  fr: 'https://support.pix.org',
+  nl: 'https://pix.org/nl-be/support',
+};
+const translations = {
+  en: enTranslations,
+  es: esTranslations,
+  fr: frTranslations,
+  nl: nlTranslations,
+};
+
+function _getMailerConfig(locale) {
+  switch (locale) {
+    case FRENCH_SPOKEN:
+    case SPANISH_SPOKEN:
+    case ENGLISH_SPOKEN:
+    case DUTCH_SPOKEN:
+      return {
+        homeName: PIX_HOME_NAME_INTERNATIONAL,
+        homeUrl: PIX_HOME_URL_INTERNATIONAL[locale] ?? PIX_HOME_URL_INTERNATIONAL.en,
+        pixOrgaHomeUrl: PIX_ORGA_HOME_URL_INTERNATIONAL,
+        pixCertifHomeUrl: PIX_CERTIF_HOME_URL_INTERNATIONAL,
+        pixAppConnectionUrl: PIX_APP_CONNECTION_URL_INTERNATIONAL[locale] ?? PIX_APP_CONNECTION_URL_INTERNATIONAL.en,
+        helpdeskUrl: PIX_HELPDESK_URL_INTERNATIONAL[locale] ?? PIX_HELPDESK_URL_INTERNATIONAL.en,
+        displayNationalLogo: false,
+        translation: translations[locale],
+      };
+    default:
+      return {
+        homeName: PIX_HOME_NAME_FRENCH_FRANCE,
+        homeUrl: PIX_HOME_URL_FRENCH_FRANCE,
+        pixOrgaHomeUrl: PIX_ORGA_HOME_URL_FRENCH_FRANCE,
+        pixCertifHomeUrl: PIX_CERTIF_HOME_URL_FRENCH_FRANCE,
+        pixAppConnectionUrl: PIX_APP_CONNECTION_URL_FRENCH_FRANCE,
+        helpdeskUrl: HELPDESK_FRENCH_FRANCE,
+        displayNationalLogo: true,
+        translation: frTranslations,
```

Comment:
Je trouve qu'il serait mieux d'utiliser directement l'objet construit `translations`.

```suggestion
        translation: translations.fr,
```